usecases/sai-ptf/README.md

```diff
@@ -10,6 +10,8 @@ SAI Challenger has capability to run these tests by setting up proper test envir
 ```
 git submodule update --init --recursive
 cp usecases/sai-ptf/ptf-conftest.py usecases/sai-ptf/SAI/ptf/conftest.py
+cp usecases/sai-ptf/patches/sai-base-test-fix-for-saivs.patch usecases/sai-ptf/SAI/
+cd usecases/sai-ptf/SAI/ && git checkout c80ff330 && patch -p0 < sai-base-test-fix-for-saivs.patch && cd -
```

Comment:
We should be on the right SAI commit after execution of L11. Please double check.

Reply:
Yes, sure. Fixed. Thanks!